src/psyclone/tests/domain/lfric/lfric_psyir_test.py

```diff
@@ -447,16 +448,16 @@ def test_vector_fields(symbol, parent_symbol, dims, attribute_map):
 def test_add_lfric_precision_symbol():
```

Comment:
This test should be moved to `lfric_symbol_table_test.py` now.

Reply:
Indeed.